packages/paste-website/src/pages/components/combobox/index.mdx

```diff
@@ -199,7 +246,7 @@ In the example below the value of the Combobox is stored in a piece of our appli
   {controlledComboboxExample}
 </LivePreview>
 
-#### useCombobox state hook
+### useCombobox state hook
```

Comment:
Same comment here as above about removing the ExternalLinkIcon from stateHookCombobox